utils/hct/hctdb.py

```diff
@@ -1727,6 +1730,23 @@ def populate_dxil_operations(self):
         self.set_op_count_for_version(1, 5, next_op_idx)
         assert next_op_idx == 216, "216 is expected next operation index but encountered %d and thus opcodes are broken" % next_op_idx
 
+        self.add_dxil_op("CreateHandleFromHeap", next_op_idx, "CreateHandleFromHeap", "create resource handle from heap", "v", "ro", [
+            db_dxil_param(0, "res", "", "result"),
+            db_dxil_param(2, "i32", "index", "heap index")])
```

Comment:
Need nonuniformresourceindex argument bool?

Reply:
Plan to change nonuniformresourceindex for 6.6